modules/cnv-vm-create-nic-web.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-configuring-pxe-booting.adoc
+
+[id="cnv-vm-create-nic-web-{context}"]
+= Creating a NIC for a virtual machine
+
+Create and attach additional NICs to a virtual machine from the web console.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+. Select a virtual machine template.
+. Click *Network Interfaces* to display the NICs already attached to the virtual machine.
+. Click *Create NIC* to create a new slot in the list.
+. Fill in the *NAME*, *NETWORK*, and *MAC ADDRESS* for the new NIC.
```

Comment:
"NETWORK" must be in the name of the one defined in the NetworkAttachmentDefinition (which is described in modules/cnv-connecting-resource-bridge-network.adoc).

Reply:
Following modular documentation principles, the module itself should not contain references to the cnv-connecting-resource-bridge-network file. A module must stand on its own. :) 

In the assembly (cnv-attaching-vm-multiple-networks.adoc), there is already the following note before the cnv-vm-create-nic-web.adoc module:

[NOTE]
====
In the next section, you must define a vNIC that refers to the bridge
network by selecting the NetworkAttachmentDefinition object you created in
the previous section.
====

Does that seem sufficient to you?

Reply:
I don't think this note is sufficient. I think it should be explicitly noted here, in order to be fully clear.
You don't have to reference the cnv-connecting-resource-bridge-network file, but rather add a note very similar to the one you presented here. Something like:
```
[NOTE]
====
NETWORK must refer to the bridge network name from the
NetworkAttachmentDefinition object you created in
the previous section.
====
```

Reply:
Unfortunately, I can't even say "the previous section" because this module could potentially be used anywhere, and it has to make sense no matter where it is used. 

If you think the current phrasing is unclear, I can change the existing note so that it specifically mentions *NETWORK* like the NIC module does. Do you think that would help?

This is the same link I provided earlier, but I'm pasting it here so you can see where the note goes (above "Creating a NIC for a virtual machine"): http://file.bos.redhat.com/pousley/042919/cnv-127/cnv/cnv-attaching-vm-multiple-networks.html

Reply:
Yes, good idea. I agree that if you specify NETWORK in this note, that would clarify what the user should assign for NETWORK.

Reply:
Great :) I changed it in https://github.com/openshift/openshift-docs/pull/14351/commits/590c5e7c9d207998764b3ad260d8d3a2848da5ce - let me know if the new note works for you. Thanks!

Reply:
Great! Thanks for the modification.